subscription-manager.spec

```diff
@@ -100,6 +100,7 @@
 %endif
 
 %global use_dnf (%{with python3} && (0%{?fedora} || (0%{?rhel}))) || (0%{?rhel} >= 7)
+%global use_libdnf (0%{?fedora} >= 29 || 0%{?rhel} >= 8)
 %global use_yum (0%{?rhel} && 0%{?rhel} <= 7)
```

Comment:
Not a change you made, but this condition overlaps with use_dnf when 0%{?rhel} == 7

Reply:
Yeah, we provide RPM with dnf plugin (it is available in some experimental repository for testing new features on rhel7).